components/Footer/Footer.js

```diff
@@ -3,74 +3,116 @@ import Link from 'next/link';
 import classNames from 'classnames';
 import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
 import SocialMedia from 'components/SocialMedia/SocialMedia';
+import Button from 'components/_common_/Button/Button';
 import styles from './Footer.css';
 
 function Footer() {
   return (
     <footer className={styles.Footer}>
-      <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
-        <a className={styles.email} href="mailto:contact@operationcode.org">
-          contact@operationcode.org
-        </a>
-
-        <SocialMedia />
-      </div>
-
-      <div className={classNames(styles.footerGrouping, styles.logoGrouping)}>
-        <img src="/static/images/icons/operation_code_logo.svg" alt="Operation Code Logo" />
+      <div className={classNames(styles.footerWrapper, styles.row)}>
+        <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
+          <div className={classNames(styles.logoGrouping)}>
+            <img src="/static/images/operation_code_logo.png" alt="Operation Code Logo" />
```

Comment:
```javascript
import { s3 } from 'common/constants/urls';

// ...

<img src={`${s3}branding/logos/small-blue-logo.png`} alt="Operation Code Logo" />
```

Reply:
I will remove the png that I added since there's already a hosted image.